server/protocols/webadmin/src/main/java/org/apache/james/webadmin/routes/UserRoutes.java

```diff
@@ -39,10 +40,9 @@
 
 public class UserRoutes implements Routes {
 
-    private static final String USER_NAME = ":userName";
-    private static final String EMPTY_BODY = "";
     private static final Logger LOGGER = LoggerFactory.getLogger(UserRoutes.class);
 
+    public static final String USER_NAME = ":userName";
```

Comment:
I use it in MailboxRoutes to reuse the constants used to access users as mailboxes are attached to a user.